OWNERS

```diff
@@ -1,14 +1,13 @@
 # approval == this is a good idea /approve
 approvers:
-  - njhale
```

Comment:
Add @dinhxuanvu?
Does it make any difference if I'm in lgtm-ers and approvers?

Reply:
There are people who are on both groups which just means you can lgtm and approve. That's all.